tsl/test/sql/continuous_aggs_usage.sql

```diff
@@ -40,11 +40,7 @@ SELECT ts, 'device_2', (EXTRACT(EPOCH FROM ts)) from generate_series('2018-12-01
 --Initially, it will be empty.
 SELECT * FROM device_summary;
 
---Normally, the continuous view will be updated automatically on a schedule but, you can also do it manually.
---We alter max_interval_per_job too since we are not using background workers
-ALTER MATERIALIZED VIEW device_summary SET (timescaledb.max_interval_per_job = '60 day');
-SET timescaledb.current_timestamp_mock = '2018-12-31 00:00';
-REFRESH MATERIALIZED VIEW device_summary;
+CALL refresh_continuous_aggregate('device_summary', NULL, '2018-12-30 22:00');
```

Comment:
If you look at the output log instead, you will see that this is the value that the previous materializer ended at. So, that one would start form "beginning of time" and end at this date.

Reply:
Could you add a comment so that it is clear that it is intentional? There are quite frequently "mysterious values" in the test cases that is hard to understand if they're relevant or not when there are changes after a patch.